src/cli.ts

```diff
@@ -32,8 +34,11 @@ export const run = (argv: string[]) => new Promise<number>((resolve) => {
   const {process: processKarmaArgs} = require('karma/lib/cli');
 
   console.info(
-      `  --proxyFile <path>   Default is ./karma.proxy.js\n` +
-      `  --proxyPort <port>   Default is 9876\n`);
+      `  --proxyFile <path>      ${
+          karmaProxyConfigFile || 'Default is ./karma.proxy.js'}\n` +
+      `  --proxyHost <hostname>  ${
+          proxyHostOption || 'Default is localhost'}\n` +
+      `  --proxyPort <port>      ${proxyPortOption || 'Default is 9876'}\n`);
```

Comment:
Thought it was useful to see the currently provided value if provided otherwise the default.